packages/webdriverio/package.json

```diff
@@ -76,9 +76,11 @@
     "@wdio/utils": "9.0.0-alpha.0",
     "archiver": "^7.0.0",
     "aria-query": "^5.0.0",
+    "cheerio": "^1.0.0-rc.12",
```

Comment:
Seems like this is pretty much jquery, why not use the newly release jquery release directly instead?

Reply:
It just looks like jQuery but is actually a completely different package. `cheerio` is an HTML parser that allows to traverse and modify HTML within Node.js. We can't use jQuery as it requires a DOM API to be available which we don't have in Node.js.